docs/src/manual/solutions.md

```diff
@@ -625,9 +625,19 @@ To use the primal solution from a solve, omit the `point` argument:
 ```jldoctest feasibility
 julia> optimize!(model)
 
-julia> primal_feasibility_report(model)
+julia> primal_feasibility_report(model; atol = 0.0)
 Dict{Any, Float64}()
 ```
+Calling [`primal_feasibility_report`](@ref) without the `point` argument is most
+useful when the solver reports that the solutionn is feasible, but you want to
+check for violated constraints at a different tolerance level.
+
+!!! warning
+    If the problem is infeasible, calling [`primal_feasibility_report`](@ref) is
+    usually the wrong thing to do, because most solvers will not return an
```

Comment:
"wrong thing to do" is too opinionated. There are plenty of scenarios where it makes sense, e.g., if you have a point in mind that you think should be feasible calling `primal_feasibility_report` is the right thing to do to debug the infeasibility.

 I'd say: To apply `primal_feasibility_report` to infeasible models, you must also provide a candidate point (solvers generally do not provide one). For diagnosing the cause of an infeasible model, see also Conflicts.